R/plotDataSetList.R

```diff
@@ -598,7 +603,10 @@ Plot.RT.Histogram.DataSetList <- function(dsList, ftarget, plot_mode = 'overlay'
       IOH_plot_ly_default(x.title = "function evaluations", y.title = "runs")
     })
   }
-
+  if (use.equal.bins){
+    res1 <- hist(get_RT_sample(dsList, ftarget, output = 'long')$RT, breaks = nclass.FD, plot = F)
```

Comment:
nclass.FD is applied on the whole data set (to all algorithms) here to get an universal bin size, right?

Reply:
Yes